docs/tutorials/autogluon-cloud.md

```diff
@@ -1,2 +1,189 @@
+It should look similar to this: `INFO:sagemaker:Creating training-job with name: ag-cloudpredictor-1673296750-47d7`.
+If for some reason, you don't have access to the log as well. You can try go to sagemaker console directly and find the ongoing training job and its corresponding job name.
+
+```{.python}
+another_cloud_predictor = TabularCloudPredictor(cloud_output_path='YOUR_S3_BUCKET_PATH')
+another_cloud_predictor.attach_job(job_name="JOB_NAME")
+```
+
+The reattached job will no longer give live stream of the training job's log. Instead, the log will be available once the training job is finished.
+
+## Endpoint Deployment and Real-time Prediction
+If you want to deploy a predictor as a SageMaker endpoint, which can be used to do real-time inference later, it is just one line of code:
+
+```{.python}
+cloud_predictor.deploy(
+    instance_type="ml.m5.2xlarge",  # Checkout supported instance and pricing here: https://aws.amazon.com/sagemaker/pricing/
+    wait=True  # Set this to False to make it unblocking call
+)
+```
+
+Optionally, you can also attach to a deployed endpoint:
+
+```{.python}
+cloud_predictor.attach_endpoint(endpoint="ENDPOINT_NAME")
+```
+
+To perform real-time prediction:
+
+```{.python}
+result = cloud_predictor.predict_real_time(
+    'test.csv',  # can be a dataframe as well
+    instance_type="ml.m5.2xlarge",  # Checkout supported instance and pricing here: https://aws.amazon.com/sagemaker/pricing/
+    wait=True  # Set this to False to make it unblocking call
+)  # result will be a pandas dataframe
+```
+
+Make sure you clean up the endpoint deployed by:
+
+```{.python}
+cloud_predictor.cleanup_deployment()
+```
+
+## Batch Inference
+When minimizing latency isn't a concern, then the batch inference functionality may be easier, more scalable, and cheaper as compute is automatically terminated after the batch inference job is complete.
+
+A general guideline is to use batch inference if you need to get predictions less than once an hour and are ok with the inference time taking 10 minutes longer than real-time inference (due to compute spin-up overhead).
+
+To perform batch inference:
+
+```{.python}
+cloud_predictor.predict(
+    'test.csv',  # can be a dataframe as well and the results will be stored in s3 bucket
+    instance_type="ml.m5.2xlarge",  # Checkout supported instance and pricing here: https://aws.amazon.com/sagemaker/pricing/
+    wait=True  # Set this to False to make it unblocking call
+)
+cloud_predictor.download_predict_results(
+    job_name=None  # download the most recent finished batch inference results to your local machine. Specify the job name to download a specific batch inference job's results.
+    save_path="PATH"  # If not specified, CloudPredictor will create one.
+)
+```
+
+## Retrieve CloudPredictor Info
+
+To retrieve general info about a `CloudPredictor`
+
+```{.python}
+cloud_predictor.info()
+```
+
+It will output a dict looks similar to this:
```

Comment:
`looking` or `that looks` is grammatically correct. Alternatively, completely remove `looks` and just be `It will output a dict similar to this:`